modules/integration/tests-integration/tests-resources/tests-resource-registry/src/test/java/org/wso2/greg/integration/resources/server/mgt/test/RegistryConfiguratorTestCase.java

```diff
@@ -194,6 +196,19 @@ private void increaseSearchIndexStartTimeDelay () throws Exception {
             AXIOMXPath xpathExpression3 = new AXIOMXPath("/wso2registry/indexingConfiguration/indexerPoolSize");
```

Comment:
It might be better to move this xpath expression variable to a constant to avoid creating n number of objects in runtime.